wa/workloads/geekbench/__init__.py

```diff
@@ -407,11 +388,16 @@ class GeekbenchCorproate(Geekbench):
     is_corporate = True
     requires_network = False
 
-    versions = ['4.1.0']
-
-    # The activity name for this version doesn't match the package name
-    activity = 'com.primatelabs.geekbench.HomeActivity'
-    package = 'com.primatelabs.geekbench4.corporate'
+    versions = {
+        '5.0.0': {
+            'package': 'com.primatelabs.geekbench4.corporate',
+            'activity': 'com.primatelabs.geekbench.HomeActivity',
+        },
+        '4.1.0': {
+            'package': 'com.primatelabs.geekbench4.corporate',
+            'activity': 'com.primatelabs.geekbench.HomeActivity',
+        },
+        }
```

Comment:
If the package name and activity do not differ then the extra version could just be appended to the versions list, however this is going to be changed to dictionary then the `version` parameter will also need to be updated so that the `allowed_values` and `default` parameters are populated correctly.